### ISSUE TITLE
Bear Proposal: Prettier

### ISSUE THREAD

**Issue by co-robo** [coala/coala-bears#2130]
https://github.com/prettier/prettier
Opened by @blazeu at [Gitter](https://gitter.im/coala/coala-bears?at=5a0da909614889d4758b6101)

**Comment by Nshul** [comment]
Can i work on this ? This would be my first time writing a bear so it may take some time.

**Comment by blazeu** [comment]
@urohit011 are you still working on this one? it's been a while

**Comment by Nshul** [comment]
Can i be assigned this??

**Comment by Makman2** [comment]
Reassigned due to inactivity.

**Comment by ishanSrt** [comment]
@Makman2 doesn't ESLintBear already use prettier ?

**Comment by blazeu** [comment]
@ishanSrt Nope.

ESLint itself can be configured (not by default) to use Prettier though.

But ESLint is only for JavaScript file, while Prettier supports additional file types as well.

**Comment by ishanSrt** [comment]
Thanks @blazeu . The PR by @Nshul says 
>This bear uses prettier to
format .js files.

thought it would be a waster to make this. Please check on that whether he is only doing it for JS.

**Comment by Nshul** [comment]
@ishanSrt sorry for late response, I had high fever so couldnt use my laptop for 2 days.
Prettier supports many file formats as provided here
[Languages supported by Prettier](https://prettier.io/docs/en/language-support.html)

My Linter Bear uses Prettier without any change so there is not any file restriction from my end, all those file formats supported by Prettier are supported by this bear. 

If you want I can the change the commit body to include all the supported formats. I have to change the tests I wrote so I can the change the commit body then. I can also change the docstring I wrote in the class of my Linter Bear file
